docker/DOCKER.md

```diff
@@ -77,56 +77,68 @@ For guidance on installation, development, deployment, and administration, see o
 
 Docker can be used to manage and run SurrealDB database instances without the need to install any command-line tools. The SurrealDB docker container contains the full command-line tools for importing and exporting data from a running server, or for running a server itself.
 
-```bash
-docker run --rm --pull always --name surrealdb -p 8000:8000 surrealdb/surrealdb:latest start
-```
-
-For just getting started with a development server running in memory, you can pass the container a basic initialization to set the user and password as root and enable logging.
+For just getting started with a development server running in memory, you can start the container with basic initialization arguments to create a root user with "root" as username and password and enable debug logging.
 
 ```bash
-docker run --rm --pull always --name surrealdb -p 8000:8000 surrealdb/surrealdb:latest start --log trace --user root --pass root memory
+docker run --rm --pull always --name surrealdb -p 8000:8000 surrealdb/surrealdb:v2 start --log debug --user root --pass root memory
```

Comment:
I'm not sure this is the best approach. If cached, the user could be stuck on `v2.0.0` for a while without noticing it too. This was an issue we experienced a while ago. It meant that we changed the Docker instructions on the [install](https://surrealdb.com/install) page to include `--pull always` like the following:

```sh
docker run --rm --pull always --name surrealdb -p 8000:8000 surrealdb/surrealdb:v2 start --log debug --user root --pass root memory
```

Perhaps we should leave it to `latest` (as this will always be the last official version), document that users can also choose a specific major (`v2`) or minor (`v2.0`), or even patch (`v2.0.4`) and always add the `--pull always` argument?

Reply:
I see your point. I was thinking more on the Docker Compose method, which I assume will be the more common. I wanted to ensure that users who tried SurrealDB in the past via Docker (like I had when I was testing this) would at least use `v2.0.0` when they deploy it in the future. This would address common issues that we get on Github and Discord resulting from users unknowingly using `1.X` while on `latest` without realizing.

I am also happy to revert to `latest` and recommending pinning a version tag instead and keeping it updated. Let me know!

Reply:
I have now changed this to use `latest` and only kept `v2` on Docker Compose.